src/cow-react/utils/fractionUtils.ts

```diff
@@ -63,3 +64,13 @@ export class FractionUtils {
     return fraction.equalTo(value) || fraction.lessThan(value)
   }
 }
+
+export function fractionToPrice(fraction: Fraction, inputCurrency: Token, outputCurrency: Token): Price<Token, Token> {
+  // Note that here the fraction shows the price in units (for both tokens). The Price class is decimals aware, so we need to adapt it
+  const adjustedFraction = adjustDecimalsAtoms(fraction, inputCurrency.decimals, outputCurrency.decimals)
+
+  return new Price({
+    quoteAmount: CurrencyAmount.fromRawAmount(outputCurrency, adjustedFraction.numerator),
+    baseAmount: CurrencyAmount.fromRawAmount(inputCurrency, adjustedFraction.denominator),
+  })
+}
```

Comment:
FractionUtils is a static class.
Move inside of it maybe?

Reply:
I will, just for coherence in the file.
But... why did we do it as static class? doesn't this just mess up with the tree-shaking when you don't need some util? 
Looks strange to me

Reply:
done 1be45b7ee89858f1061b3f5a397a733d45a32199

Reply:
I raised the same question to Sasha 😅 

He convinced me it wouldn't be a big deal and I didn't insist any further